app/features/ledger/SignAddIdentityProvider.ts

```diff
@@ -20,6 +21,36 @@ import {
 
 const INS_ADD_IDENTITY_PROVIDER = 0x2d;
 
+/**
+ * Sends the given description for signing.
+ * Uses the given p1 for length and p1 + 1 for the content.
+ */
+export async function sendDescription(
```

Comment:
This should be moved to a shared file instead of residing here, since it is used by two transactions and not just this one.